### ISSUE TITLE
Issue with override of the translations

### ISSUE THREAD

**Issue by nuovecode** [yireo/Yireo_EmailOverride#27]
Hi, I'm on Magento 1.9.2.3

I have to override the Mage_Sales.csv. After installing your extension (that works perfectly with the email templates) I copied:

app/locale/it_IT/Mage_Sales.csv

in

app/design/frontend/<PACKAGE>/<THEME>/locale/it_IT/Mage_Sales.csv

But it does'nt work. What's wrong?

Thanks!

**Comment by jissereitsma** [comment]
Hi, could you confirm that `app/design/frontend/PACKAGE/THEME` is indeed the folder name of your theme?

**Comment by nuovecode** [comment]
Yes. There are also app/design/PACKAGE/THEME/locale/it_IT/template/email/html/footer.html that works here

**Comment by jissereitsma** [comment]
And could you confirm the exact version that you have installed?

**Comment by nuovecode** [comment]
Yes 0.2.16

**Comment by jissereitsma** [comment]
Then I really don't know what could be causing it. Perhaps a conflict with another extension? Perhaps something else?

**Comment by nuovecode** [comment]
don't know, i'll investigate. It was just  to be sure I did'nt do something wrong. Thanks so much!

**Comment by nuovecode** [comment]
I may have found the problem. Mage_Sales.csv translates strings in transational email template. In your Translate.php model on line 28 there is 

```
 if ($this->isAdmin() == true) {
     return parent::_getModuleFilePath($module, $fileName);
}
```
isAdmin return true and it loads Mage_Core.csv, Without this expression it loads Mage_Sales.csv correctly
